public/ajax/add-password.php

```diff
@@ -0,0 +1,64 @@
+<?php
+// Import required dependencies
+include_once '../../config.php';
+// include the database connection file
+include "../../db.php";
+
+// Start session
+session_start();
+
+// When not logged in, redirect the client to the homepage
+if (!isset($_SESSION['masterpass']) && !isset($_SESSION['username'])) {
+    session_start();
+    session_destroy();
+    echo json_encode(['success' => false, 'redirect' => '/']);
+    exit;
+}
+
+// instantiate a new database object
+$database = new DataBase();
+
+// Initialize the $errorMsg variable to null.
+$errorMsg = null;
+
+// check if the form has been submitted
+if (isset($_POST['website'], $_POST['username'], $_POST['password'])) {
+    // if the form has been submitted, add the password to the database
+    // using the add_password() function of the database object
+    try {
+        $database->add_password($database->getUserId(), trim($_POST['website']), trim($_POST['username']), trim($_POST['password']));
```

Comment:
Is here or in other places no need to sanitize the POST data?

Reply:
sanitized the data